### DISCUSSION TITLE
如何设置手续费

### DISCUSSION THREAD

**Discussion by chentianqi-3** [Chia-Network/chia-blockchain#17177] [Support]
如何设置手续费，提高命中率，减少log文件报错。

**Comment by BrandtH22** [comment]
closing this discussion as it is a duplicate of https://github.com/Chia-Network/chia-blockchain/discussions/17176

---

结束此讨论，因为它是 https://github.com/Chia-Network/chia-blockchain/discussions/17176 的重复
